zoho.com.zmail_hosting.json

```diff
@@ -8,60 +8,80 @@
     "variableDescription": "",
     "syncBlock": true,
```

Comment:
If the intention of the template change was to unblock the sync flow, then pls change this setting to false (or remove).